### PR TITLE
bar-xft: initial commit.

### PR DESCRIPTION
**Pull request by christianlask**
This expression builds a fork of the `bar` by lemonboy with
xft-support.

### CODE REVIEW

File: pkgs/applications/window-managers/bar/xft.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      rev = "020a3e1848ce03287886e9ff80b0b443e9aed543";
+      sha256 = "1xzs37syhlwyjfxnk36qnij5bqa0mi53lf1k851viw4qai2bfkgr";
+    };
+  
+    buildInputs = [ libxcb libXft perl ];
+  
+    prePatch = ''sed -i "s@/usr@$out@" Makefile'';
+  
+    meta = {
+      description = "A lightweight xcb based bar with XFT-support.";
```
Comment (pSub) [inline pkgs/applications/window-managers/bar/xft.nix]: Descriptions should not end with a period. See the nixpkgs manual.

**Comment by hiberno** [comment]
@pSub Should be fixed now. Sorry.

**Comment by jagajaga** [comment]
Please rename commit's name following our guide in `CONTRIBUTION.md`. And don't forget to squash commits into one.

**Comment by hiberno** [comment]
@jagajaga There's quite some stuff to learn and think of. :D I hope everything's just fine now?

**Pull request merged**